include/pybind11/attr.h

```diff
@@ -12,7 +12,7 @@
 
 #include "cast.h"
 
-NAMESPACE_BEGIN(pybind11)
+NAMESPACE_BEGIN(PYBIND11_NAMESPACE)
```

Comment:
Ah, I guess that's why `NAMESPACE_BEGIN/END` are the only macros not prefixed with `PYBIND11` :)